src/server/utils/loadModules.js

```diff
@@ -25,9 +26,17 @@ import { setClientModuleMapCache } from './clientModuleMapCache';
 import { updateCSP } from '../plugins/csp';
 import addBaseUrlToModuleMap from './addBaseUrlToModuleMap';
 
+let cashedModuleMapHash;
+
 const loadModules = async () => {
   const moduleMapResponse = await fetch(process.env.HOLOCRON_MODULE_MAP_URL);
   const moduleMap = addBaseUrlToModuleMap(await moduleMapResponse.json());
+
+  const moduleMapHash = hash(moduleMap);
+  if (cashedModuleMapHash && cashedModuleMapHash === moduleMapHash) {
+    return {};
+  }
+  cashedModuleMapHash = moduleMapHash;
```

Comment:
```suggestion
let cachedModuleMapHash;

const loadModules = async () => {
  const moduleMapResponse = await fetch(process.env.HOLOCRON_MODULE_MAP_URL);
  const moduleMap = addBaseUrlToModuleMap(await moduleMapResponse.json());

  const moduleMapHash = hash(moduleMap);
  if (cachedModuleMapHash && cachedModuleMapHash === moduleMapHash) {
    return {};
  }
  cascedModuleMapHash = moduleMapHash;
```

Minor spelling issue